### ISSUE TITLE
TurfPy Concave throws ZeroDivisionError: float division by zero for concave hull

### ISSUE THREAD

**Issue by Khaalidi** [omanges/turfpy#99]
```
import json
from turfpy.transformation import concave
from geojson import FeatureCollection, Feature, Point
import sys
import pandas as pd

def get_points_csv(file_path):
	df = pd.read_csv(file_path)
	long_coords, lat_coords = df['Longitude'].tolist() , df['Latitude'].tolist()
	points = []

	for i in range(0, len(long_coords)):

		points.append((long_coords[i],lat_coords[i]))
	return points

points = get_points_csv(sys.argv[1])

fc = []
for p in points:
	fc.append(Feature(geometry=Point(p)))
	# print(p)

ch = concave(FeatureCollection(fc), alpha=100)
print(json.dumps(ch, indent=2, sort_keys=True))
```

```
Traceback (most recent call last):
  File "C:\Users\Khaalidi\Desktop\area_turfpy.py", line 26, in <module>
    ch = concave(FeatureCollection(fc), alpha=100)
  File "C:\Users\Khaalidi\AppData\Local\Programs\Python\Python39\lib\site-packages\turfpy\transformation.py", line 418, in concave
    concave_hull, edges = _alpha_shape(points, alpha)
  File "C:\Users\Khaalidi\AppData\Local\Programs\Python\Python39\lib\site-packages\turfpy\transformation.py", line 340, in _alpha_shape
    circum_r = a * b * c / (4.0 * area)
ZeroDivisionError: float division by zero
```
The coordinates csv : [coord.csv](https://mega.nz/file/T5Z2HZJb#IPTF3CrEZFEPaCYQCAIA6icfLSsr8gFgeMqVIUvfUJg)
The features Collection is: [geo.txt](https://mega.nz/file/rsJmUB5A#jTcEKveqwnM0dLvgzBGUeYfD2Gc4oyYTunhYAKNPnqc)

**Comment by omanges** [comment]
@Khaalidi  Thanks for pointing out the issue, we will take up this issue in next release.

**Comment by omanges** [comment]
@Khaalidi Fixed in release v0.0.7
